lib/projects.rb

```diff
@@ -0,0 +1,33 @@
+require_relative "project_printer"
+
+class Projects
+  attr_reader :projects, :project_printer 
+  
+  def initialize(project_printer)
+    @projects = []
+    @project_printer = project_printer
+  end
+
+  def projects_empty?
```

Comment:
The naming on these is kind of awkward. You have a class and related method in the runner that is projects, so duplicating that in each of these methods is funky. If this were a data class and more general, then it would be appropriate and likely an extraction smell to get to this projects class.